docs/blog/posts/why_mesop.md

```diff
@@ -0,0 +1,53 @@
+
+- **Simplicity:** You can use your existing Python knowledge to build UIs quickly and intuitively since components are just functions.
+- **Maintainability:** Complex UIs become easier to manage and understand by breaking them down into smaller, focused components.
+- **Modularity:** Components are self-contained, enabling easy reuse within a project or across different projects.
+
+Here's an example of a reusable icon button component:
+
+```python
+def icon_button(*, icon: str, label: str, tooltip: str, on_click: Callable):
+  """Icon button with text and tooltip."""
+  with me.content_button(on_click=on_click):
+    with me.tooltip(message=tooltip):
+      with me.box(style=me.Style(display="flex")):
+        me.icon(icon=icon)
+        me.text(
+          label, style=me.Style(line_height="24px", margin=me.Margin(left=5))
+        )
+
+```
+
+## Flexibility through Layered Building Blocks
+
+Mesop provides a range of UI building blocks, from low-level [native components](https://google.github.io/mesop/guides/components/#native-components) to high-level components.
+
+- Low-level components: like [box](../../components/box.md), offer granular control over layout and styling. They empower you to create custom UI elements through flexible layouts like flexbox and grid.
+- High-level components: like [chat](../../components/chat.md), are built from low-level components and provide ready-to-use elements for common use cases, enabling rapid development.
+
+This layered approach makes deep customization possible — all in Python. This means that if you want to customize the chat component, you can fork the [chat implementation](https://github.com/google/mesop/blob/main/mesop/labs/chat.py) because it's written entirely in Python using Mesop's public APIs.
+
+## See Mesop in Action
+
+To demonstrate the range of UIs possible with Mesop, we built a demo gallery to showcase the different kinds of applications you can build and the components that are available:
+
+<iframe class="immersive-demo" src="https://mesop-y677hytkra-uc.a.run.app/"></iframe>
+
+ The [demo gallery](https://mesop-y677hytkra-uc.a.run.app/) itself is a Mesop app and implemented in a few hundred lines of Python code. It demonstrates how Mesop can be used to create polished, bespoke UIs in a maintainable way, going beyond the limitations of typical demo apps.
+
+## Try Mesop
+
+If this sounds intriguing, read the [Getting Started guide](../../getting_started.md) and build your first Mesop app today. We're excited to see what you build – from prototypes to full-fledged applications. Mesop aims to provide a flexible and delightful UI development experience for Python developers. Share your feedback and contribute as we continue developing [Mesop on GitHub](https://github.com/google/mesop).
```

Comment:
I feel like this last paragraph could be a approved from a style/voice point of view. It reads a bit too "sale-sy / marketing"

Specifically these phrases:

- "If this sounds intriguing..."
- "build your first Mesop app today"
- "delightful" - Probably should keep "delightful". On one hand, building Mesop apps is pretty "delightful". On the other hand, it reads sort of "marketing" speak. At the same time, it's sort of part of your main documentation phrasing, so iterating the phrase here can work for emphasizing your tag line for Mesop.

Reply:
OK, I've made it less sales-y